gobase/utils/converter.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import "strings"
+
+func ConvertToPluralNoun(noun string) string {
+	suffixList := []string{
+		"s", "x", "z", "ch", "sh",
+	}
+	for _, s := range suffixList {
+		if strings.HasSuffix(noun, s) {
+			return noun + "es"
+		}
+	}
+	if strings.HasSuffix(noun, "y") {
+		return noun[:len(noun)-1] + "ies"
+	}
+
+	irregularNoun := map[string]string{
```

Comment:
should put it on specific constants file

Reply:
Resolved in PR https://github.com/wincentrtz/gobase/pull/5